docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -2002,12 +2008,88 @@ The default is `INFO`, which is recommended in production.
 <Tabs>
   <TabItem label="values.yaml">
   ```yaml
-  logLevel: DEBUG
+  log:
+    level: DEBUG
   ```
   </TabItem>
   <TabItem label="--set">
   ```code
-  $ --set logLevel=DEBUG
+  $ --set log.level=DEBUG
```

Comment:
I think we should make sure that shell prompts are used consistently across code snippets. Some include `$` and some do not.

Reply:
Agreed! I don't think it's a good idea to fix all of them as part of this particular PR (they should be done separately as part of a wider cleanup) but I have removed this one weird instance.